kano_updater/scenarios.py

```diff
@@ -341,3 +344,24 @@ def beta_210_to_beta_220(self):
             logger.error(
                 "Could not award Computer Commander badge, import error"
             )
+
+    def beta_220_to_beta_230(self):
+        os.system('sudo adduser {} i2c'.format(get_user_unsudoed()))
```

Comment:
We probably need all of the linux users on the kit to be added to this group and change it so new users are added as well.